.travis.yml

```diff
@@ -47,7 +47,7 @@ matrix:
     - env:
       - NAME="FULL 3.8"
```

Comment:
We would need to change the name here too; however the FULL test worked so we can switch it back. Please do merge #122 and #124 together. Either PR is fine.

Reply:
I don't know how to merge two PR with github web interface. please redo your own PR, if you want this. Glad to know Python-3.8 will work

Reply:
Closing the previous one and recommitting to this branch will work as well. I don't think you can merge through the GUI either.